data/human/human missions.txt

```diff
@@ -9341,3 +9342,799 @@ mission "Timothy Radrickson 7: Grave"
+			choice
+				`	"Thank you."`
+					goto "gotta go"
+				`	"Is it wise for you and your child to be left alone in a place like Haven?"`
+
+			`	She laughs. "After surviving on Hope and Chiron? Haven will be a cakewalk."`
+				goto end
+
+			label "gotta go"
+			`	"No, it is my appreciation that has to be given. You have done what few would have the conviction to do."`
+
+			label end
+			`	She takes her child in one hand, and the briefcase in the other, and walks off your ship.`
+
+
+event "pirateildy"
+
+
+mission "Ice Queen 7"
+	landing
+	name "Meet Ildy on <planet>"
+	description "See what Ildico needs on <destination>."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination "Haven"
+	to offer
+		has "event: pirateildy"
+	on offer
+		dialog
+			`Your inbox blinks with a new message from Ildico from Haven:`
+			``
+			`	"It's been too long, and I could use your help. Please, come to <planet>.`
+			`				-your friend`
+			`					Ildy"`
+
+	on complete
+		dialog
+			`Even from orbit, Haven looks ugly, and as you approach the planet it doesn't get any better. Upon landing you're alerted Ildico is waiting for you in the spaceport.`
+
+
+mission "Ice Queen 8"
+	name "Defeat Ildico's rivals"
+	description "Ildico has requested you to destroy a fleet of Marauder ships near the <system> system. Once you're done, return to <planet>."
+	source "Haven"
+	to offer
+		has "Ice Queen 7: done"
+	on offer
+		conversation
+			`You arrive at the address Ildico gave you; a large warehouse on the edge of the spaceport. When you step into the dimly lit building, your eyes struggle to adjust, but when you're able to see clearly you find yourself face-to-face with a group of dirty burly men and women leveling a half-dozen weapons right at you.`
+			`	"Looks like someone's lost," a voice rises from the group, though you can't tell who's speaking.`
+			`	"They look right fancy. Good thing we found them."`
+			`	"Whaddya say, Frank, kill 'em now? Or keep 'em for a bit?"`
+			choice
+				`	"Ildy?"`
+				`	"Ildy asked me to come here."`
+				`	"Ildy, help!"`
+				`	(Grab your sidearm and shoot your way out.)`
+					goto why?
+
+			`	The band of cut-throats looks at you, confused, and then most of them lower their weapons.`
+			`	"Boss," one of them hollers, "this stranger says they're here for you, I think."`
+			`	Footsteps echo on the opposite side of the warehouse as someone approaches.`
+			`	One of the men starts laughing. "Ildy?! Haha, never thought to call her that. Hey Ildy! How come you never told us to call you that? Or that some kind of special pet name?"`
+			`	The man flashes an unsavory grin at you, with his rotting teeth on full display.`
+			`	In the dim light the footsteps materialize into a woman in her mid-thirties with hair darker than vacuum. Ildico Drifa gives you a warm smile, then raises up her hand and fires her side-arm.`
+			choice
+				`	(Duck.)`
```

Comment:
Again, try not to have single choices if possible.

Reply:
I disagree with the blanket reasoning for this approach (I know it's a style guide thing). In nearly all video games, even ones where you appear to have endless choices, there are times where you really only one have one. You can only go through one door, or you can only walk in one direction, or you can only have one interaction at a crucial moment. The better games do not automate these moments for you, but force you to make the only choice you're allowed to make. To do it for you is to make you passive, is to regrade the medium for interactive to acted upon.

That said, I agree more options are better here, and have done as requested.